Explorer/Assets/DCL/Multiplayer/SDK/Systems/GlobalWorld/ICharacterDataPropagationUtility.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Arch.Core;
+using DCL.Profiles;
+using SceneRunner.Scene;
+
+namespace DCL.Multiplayer.SDK.Systems.GlobalWorld
+{
+    public interface ICharacterDataPropagationUtility
```

Comment:
Do we need this interface?

Reply:
No harm to rely on abstractions considering it's created from the container. I would go without interface if it was instantiated directly in one class and not shared between multiple ones